### PR TITLE
Created a Timer

### PR DESCRIPTION
**Pull request by Beggie98**
Added a java class Timer

### CODE REVIEW

File: Test/src/main/java/java/Timer.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            if (i == 1 || i == 0){
+                minutes = " minute";
+            }
+            for (int j = 59; j >= 0 ; j--) {
+                String seconds = " seconds";
+                if (j == 1 || j == 0){
+                    seconds = " second";
+                }
+                System.out.print("\r" + i + minutes + " and " + j + seconds);
+                Thread.sleep(1000);
```
Comment (SHOKHZODJURAEV) [inline Test/src/main/java/java/Timer.java:24]: please don't use Thread.sleep function.

File: Test/src/main/java/java/Timer.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            }
+        }
+
+        System.out.println("====================================");
+        System.out.println("|           TIME IS UP             |");
+        System.out.println("====================================");
+
```
Comment (SHOKHZODJURAEV) [inline Test/src/main/java/java/Timer.java:31]: please make more specific to understand.

**Comment by Beggie98** [comment]
Working on it

File: Test/src/main/java/java/Timer.java (hunk 1 above)
Comment (Beggie98) [reply Test/src/main/java/java/Timer.java:24]: working on it

File: Test/src/main/java/java/Timer.java (hunk 2 above)
Comment (Beggie98) [reply Test/src/main/java/java/Timer.java:31]: working on it

**Review by SHOKHZODJURAEV — APPROVED** [review]
looks good to me.

**Comment by Sherlock9898** [comment]
Can you check your ccode again ?

**Pull request merged**